modules/userinterfacegl/src/processors/camerawidget.cpp

```diff
@@ -656,7 +645,7 @@ vec3 CameraWidget::rotationAxis(RotationAxis rot, bool alignToObject, const Came
             case RotationAxis::Yaw:
                 return cam.up;
             case RotationAxis::Pitch:
-                return cam.right;
+                return glm::cross(cam.dir, cam.up);
```

Comment:
Are both `cam.dir` and `cam.up` always normalized? If not, the result should be normalized.